src/api/app/assets/javascripts/webui2/request.js

```diff
@@ -152,3 +152,31 @@ function requestAddReviewAutocomplete() { // jshint ignore:line
     max: 50
   });
 }
+
+function setupActionLink() { // jshint ignore:line
```

Comment:
i wonder if those "general" changes shouldnt be done in the end of the deliverable, to avoid unnecessary merge requests. Done in a separated commit

Reply:
Not sure what you mean tbh. If the code isn't needed we shouldn't add it.

Reply:
nevermind, this function came back in the last merge conflict

Reply:
I think it's fine to do such things on the fly. Otherwise they might get forgotten. And merge conflicts are something we have to live with. It happens sometimes.